packages/instrumentation-remix/src/instrumentation.ts

```diff
@@ -231,6 +231,9 @@ export class RemixInstrumentation extends InstrumentationBase {
           );
           return originalResponsePromise
             .then((response) => {
+              if (response.status === 500) {
```

Comment:
Can you move this block to be inside the `addResponseAttributesToSpan` function so that all "responses" get the attribute?

Also, you should really check for any `response.status >= 500 && response.status <= 599` to be more complete. Any `5xx` status is technically a server error.